src/main/java/org/fundacionjala/coding/jimmy/Kata2.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.jimmy;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+/**
+ * This kata was performed by Jimmy Romero.
+ */
+public final class Kata2 {
+
+    /**
+     * Constructor method.
+     */
+    private Kata2() {
+    }
+
+    private static final int MINIMUN_LENGTH_OF_WORD = 2;
+
+    /**
+     * @param stringWords is the complete string which is going to change its inner.
+     * @return the inner of the word sorted.
+     */
+    public static String sortInnerContent(String stringWords) {
+        return Arrays.stream(stringWords.split(" ")).map(w -> sortWord(w)).collect(Collectors.joining(" "));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)